src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json.schemastore.org/vs-2017.3.host",
   "name": {
-    "text": "Web API",
+    "text": "API",
```

Comment:
This won't change what gets displayed in the One ASP.NET dialog. The "text" value in the host file is only a fallback in case the package and resource id specified can not be loaded.

We have two options here:
1. I can make a resource change in the One ASP.NET Dialog to change the name of the template to "API". This would only apply to Dev 15.6 Preview 2+. This would impact both 1.x and 2.0 templates displayed,

2. Since in this case, API gets localized in all languages (I just confirmed in RUS and FRA),  you could simply remove the package and id properties here, in that case, in absence of those properties, VS will just use what is in the text field for all languages.

Reply:
@Eilon I like the first option. Just to be consistent with other templates and ensure that it's always localized correctly.

Reply:
@Eilon if you are fine with this, I'll get this checked in for our next WTE build and insertion for 15.6.
@mlorbetske FYI

Reply:
Fine with me.

Reply:
Thanks, will get this in the next build.